.vitepress/sidebars/guides.ts

```diff
@@ -5,84 +5,53 @@ export const guidesSidebar: DefaultTheme.SidebarItem[] = [
     text: "Guides",
     items: [
       {
-        text: "Introduction",
-        link: "/guides/",
+        text: "Importing Caido's CA Certificate",
```

Comment:
I would add the "Install" item as the first item. Otherwise there's no intuitive way to go back to the /guides/index page.